azkaban-common/src/main/java/azkaban/executor/ConnectorParams.java

```diff
@@ -36,6 +36,7 @@
   public static final String RELOAD_JOBTYPE_PLUGINS_ACTION = "reloadJobTypePlugins";
   public static final String ACTIVATE = "activate";
   public static final String DEACTIVATE = "deactivate";
+  public static final String GET_EXECUTOR_STATUS = "getExecutorStatus";
```

Comment:
nit: same comment again. Why not `GET_STATUS` and `getStatus` ? The API is being called on the executor itself. Personally, I prefer succinct names which not include redundant information.